### DISCUSSION TITLE
Health system and census

### DISCUSSION THREAD

**Discussion by dimitri-tikhomirov** [Keriew/augustus#566] [Ideas]
In vanilla game city health is based on the fraction of people that are serviced by clinics and clinics only. Other buildings from the health panel (baths, barbers, and even hospitals) don't contribute to health at all. Perhaps the health rating should be more complex?

In vanilla raising health from low levels to a medium level is beneficial, because it stops certain bad events, like diseases, from happening in the city. However, having a high health level is not really beneficial as opposed to having a medium health level. Quite the contrary, it has a big downside of having a smaller workforce percentage. An explanation how this works in the game:

Workforce depends on census — 60% of plebeians from age 20 to 49 work. Census in turn depends on city health, because city health determines death rates in different age groups. Generally, higher health levels lead to older population and lower workforce percentage.

Perhaps there should be some benefits for having high health? For example:

- Make health affect sentiment.
- Make census affect the culture rating. Older populations should receive a bonus to culture. I think it's realistic for older people to be more familiar with culture and tradition. This will also allow Augustus to lower bonuses to culture from building upgrades tied to desirability, which seem a little too drastic at the moment (an upgraded theater, which occupies the same 2x2 space, covers almost 2.5 times more people).

**Comment by Legionary2** [comment]
I like your ideas, and will discuss them with the development team on discord - indeed health should be tinkered with as it really makes no sense from a labor standpoint - I like your culture idea as well - as people aged, traditions and values are grown and accustomed.

**Comment by Filu2259** [comment]
Make poor health affect sentiment per house, and great health increase overall culture? I like it.
